### PR TITLE
fix: #156

### PR DESCRIPTION
**Pull request by hashue**
## Types of changes

What kind of change does this PR introduce? (check at least one)

- [ ] Breaking change
- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactoring
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Description

Issue Number: #156

### CODE REVIEW

**Comment by LumaKernel** [comment]
please rebase or merge main

**Comment by hashue** [comment]
hum… CI is still dropping…

**Comment by LumaKernel** [comment]
you may see your tests failing in your local environment now. typing `: File | ReadStream` added into main after your PR opening.

**Comment by hashue** [comment]
OK. I'll check it.

**Review by LumaKernel — CHANGES_REQUESTED** [review]
Can you add sample to show 156 is solved? I can't see whether 156 is fixed only by diff.

File: samples/file-array/@types/index.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/* eslint-disable */
+import type { ReadStream } from 'fs'
+
+export type Files = {
+  files: (File | ReadStream)[]
```
Comment (LumaKernel) [inline samples/file-array/@types/index.ts:5]: :+1:

**Review by LumaKernel — APPROVED** [review]

**Pull request merged**